### PR TITLE
Message toast

### PR DESCRIPTION
**Pull request by LucioPelinson**

### CODE REVIEW

**Comment by LucioPelinson** [comment]
Captura de Tela 2024-04-27 às 18 18 33

File: src/Core/UI/Components/Html/MessageToast/MessageToastFactory.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        
+        switch (color)
+        {
+            case BootstrapColor.Danger:
+                messageToast.Icon.IconClass = IconType.TimesCircle.GetCssClass();
+                messageToast.Title = stringLocalizer["Erro"];
+                break;
+            case BootstrapColor.Warning:
+                messageToast.Icon.IconClass = IconType.Warning.GetCssClass();
+                messageToast.Title = stringLocalizer["Aviso"];
```
Comment (gumbarros) [inline src/Core/UI/Components/Html/MessageToast/MessageToastFactory.cs]: English please :)

File: src/Core/UI/Components/Html/MessageToast/MessageToastFactory.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        messageToast.TitleMuted = DateTime.Now.ToShortTimeString();
+        messageToast.TitleColor = color;
+        messageToast.Icon = new JJIcon();
+        messageToast.Icon.CssClass = "fs-7 me-1";
+        
+        switch (color)
+        {
+            case BootstrapColor.Danger:
+                messageToast.Icon.IconClass = IconType.TimesCircle.GetCssClass();
+                messageToast.Title = stringLocalizer["Erro"];
```
Comment (gumbarros) [inline src/Core/UI/Components/Html/MessageToast/MessageToastFactory.cs]: English please :)

File: src/Core/UI/Components/Html/MessageToast/JJMessageToast.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            .WithAttribute("aria-atomic", "true")
+            .AppendDiv(header =>
+            {
+                header.WithCssClass("toast-header")
+                .Append(HtmlTag.Strong)
+                .WithCssClass($"me-auto text-{TitleColor.ToColorString()}")
+                .Append(HtmlTag.Strong, s =>
+                {
+                    s.WithCssClass($"me-auto text-{@TitleColor.ToColorString()}")
+                        .AppendComponent(Icon)
```
Comment (gumbarros) [inline src/Core/UI/Components/Html/MessageToast/JJMessageToast.cs:42]: We should check if Icon is null right?

**Comment by gumbarros** [comment]
Just these small points, nice PR, very nice idea ⭐

File: src/Core/UI/Components/Html/MessageToast/JJMessageToast.cs (hunk 3 above)
Comment (gumbarros) [reply src/Core/UI/Components/Html/MessageToast/JJMessageToast.cs:42]: No need to check, `AppendComponent `already check for null

**Review by gumbarros — APPROVED** [review]
LGTM

**Pull request merged**